src/components/PrivacyCheckbox.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {getBEMClassName} from '../utils';
+
+
+const PrivacyCheckbox = ({label, value, onChange}) => {
+  return (
+    <div className={getBEMClassName('privacy-checkbox')}>
+      <input
+        type="checkbox"
+        name="privacy"
+        id="privacy"
+        required={true}
```

Comment:
FYI - you can just use the attribute without "value" to specify booleans

```suggestion
        required
```